Xls/Reader/excel_openpyxl.py

```diff
@@ -25,7 +25,11 @@ def run(argv):
     for row in sheet.iter_rows(row_offset=int(args.start) - 1):
         current_row_read = []
         for cell in row:
-            current_row_read.append(cell.value.encode('utf-8'))
+            value = cell.value
+            if isinstance(value, (int, long)):
```

Comment:
@tier2003 
If you want to go in such details, then listen.
I call `python` executable which is convenient and cross-platform. But for our system`python` is alias for `python2.7`. New version is indeed available, but only through the non-convenient name `python3`
In this lib you can pass custom executable name, but I don't want to spend my time on using non-default cofiguration